zest/releaser/release.py

```diff
@@ -251,9 +253,9 @@ def _release(self):
             return
 
         package = self.vcs.name
-        version = self.data['version']
+        tag = self.pypiconfig.tag_format() % ({'version': self.data['version']})
         logger.info("Doing a checkout...")
-        self.vcs.checkout_from_tag(version)
+        self.vcs.checkout_from_tag(tag)
         # ^^^ This changes directory to a temp folder.
         self.data['tagdir'] = os.path.realpath(os.getcwd())
```

Comment:
I added this in the prepare function
```python
    def prepare(self):
        """Collect some data needed for releasing"""
        self._grab_version()
        self.data['tag'] = self.pypiconfig.tag_format() % (
            {'version': self.data['version']})
        self._check_if_tag_already_exists()
```
But the tests are now failing because they do not run `prepare`
Should I store the tag in `_grab_version` or in another `_grab...` function in `Basereleaser` and call it in the test?